sdk/loadtestservice/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -2,4 +2,15 @@
 
 ## 1.0.0b1 (2022-07-28)
 
+### Features Added
 - Initial version
+
+## 1.0.0b2 (2022-08-04)
+
+### Bug Fixed 
+- fixed `delete_app_components` method from `azure.developer.loadtesting.LoadTestingClient.load_test_administration.delete_app_components`
+
+### Other Chnages
+- Updated README
+- Added more Unit Tests
+- Added and refined Samples
```

Comment:
Put `1.0.0b2` ahead of `1.0.0b1`: 
![image](https://user-images.githubusercontent.com/70930885/183008213-eec693a0-10e3-4feb-b470-ecb32f0ec1bc.png)